test/src/module/command/infoTest.c

```diff
@@ -1171,9 +1173,72 @@ testRun(void)
             "            database size: 19.2MB, backup size: 8.2KB\n"
             "            repository size: 2.3MB, repository backup size: 346B\n"
             "            backup reference list: 20181119-152138F, 20181119-152138F_20181119-152152D\n"
-            "            database list: none\n",
+            "            database list: none\n"
+            "            files with page checksum error: base/16384/17000\n",
             "text - backup set requested, no db");
 
+        // Backup set requested without page checksum errors
```

Comment:
This test is not needed if you do the following:

line 1146 - change the TEST_MANIFEST_FILE to be your TEST_MANIFEST_FILE_NO_CHECKSUM_ERROR below.

Reply:
applied, thanks